app/views/kaminari/ats/_paginator.html.slim

```diff
@@ -5,9 +5,9 @@
     per_page:      number of items to fetch per page
     remote:        data-remote
     paginator:     the paginator that renders the pagination tags inside
-    klass:         additional classes for wrapper
+    class:         additional classes for wrapper
 == paginator.render do
-  nav.page-paginator aria-label="pager" class=klass
+  nav.page-paginator.mt-3 aria-label="pager" class=local_assigns[:class]
```

Comment:
Here

Reply:
i mean not only activities for positions, but also for candidates
it is different files

Reply:
This file is used by all paginators

Reply:
`/ats/candidates/activities_tab.html.slim` also contains `mt-3`. will be removed in separate pr
agreed on a call.